### ISSUE TITLE
The groot2 only support xml extension

### ISSUE THREAD

**Issue by Ovalgee** [BehaviorTree/Groot2#27]
In some environment or software, they are  only can load  special extension format file, but groot2 must open or save the xml format.

**Comment by facontidavide** [comment]
Can you rephrase? What is the problem or the intended behavior?

**Comment by Ovalgee** [comment]
The unity3d can only read .txt extension format file, but the groot2 must save btree file to .xml, I must rename these files to .txt for unity3d.

**Comment by Ovalgee** [comment]
Maybe some other software has same problem, or for customize, we can save the btree file to .ai or some other else, but the groot2 support only .xml
